src/components/organisms/Vault/vault.organism.module.scss

```diff
@@ -146,6 +146,13 @@
 
       .content {
         padding-bottom: 45px;
+
+        .balanceLabel {
+          cursor: pointer;
+          text-decoration: underline;
+          color: map-get($colors, "tiffanyBlue");
```

Comment:
Since the color is referring to the primary accent, it should get its value from the accent map.